src/actions/session.js

```diff
@@ -100,8 +100,9 @@ export function logout(): {
 }
 
 function navigateToFxA(server: string, redirect: string): NavigationResult {
-  document.location
-    .href = `${server}/fxa-oauth/login?redirect=${encodeURIComponent(redirect)}`;
+  document.location.href = `${server}/fxa-oauth/login?redirect=${encodeURIComponent(
+    redirect
+  )}`;
```

Comment:
I'm curious as to why this has been reformatted. Did you `npm install` to retrieve the latest version of *prettier*?